lerna.json

```diff
@@ -3,9 +3,19 @@
 	"version": "independent",
 	"command": {
 		"publish": {
-			"ignoreChanges": ["*.md"],
-			"message": "chore(release): publish"
+			"ignoreChanges": ["**/*.md", "**/__tests__/**"],
```

Comment:
Not a review comment but fucking turkeys on a blazing sunday afternoon why have I completely missed this option when reading lerna docs the many many times that I have I feel like such a dumbass now. 

I'm adding this to Sapphire projects right await LMFAO

Reply:
This is an actual review comment, this line has to be moved outside of the commands object. See:

https://github.com/lerna/lerna/blob/main/lerna.json